Lib/test/test_clinic.py

```diff
@@ -161,11 +161,8 @@ def test_checksum_mismatch(self):
             [clinic start generated code]*/
             /*[clinic end generated code: output=0123456789abcdef input=fedcba9876543210]*/
         """
-        err = (
-            'Checksum mismatch!\n'
-            'Expected: 0123456789abcdef\n'
-            'Computed: da39a3ee5e6b4b0d\n'
-        )
+        err = ("Checksum mismatch! "
+               "Expected '0123456789abcdef', computed 'da39a3ee5e6b4b0d'")
```

Comment:
I'm not quite sure about this, though. Perhaps the existing format is better.

Reply:
I think this is fine. The error message in `test_cli_force` is definitely better, and IIUC you can't have one without the other